cobigen-cli/index.asciidoc

```diff
@@ -68,3 +68,24 @@ For example, the following files are generated by CobiGen when using the specifi
 (8) CRUD SpringData Repository: Generates the entity repository (that contains the CRUD operations) in the data access layer.
 - `devonfw/workspaces/main/cobigenexample/core/src/main/java/com/example/application/cobigenexample/customermanagement/dataaccess/api/repo/CustomerRepository.java`{{open}}
 ====
+
+Create the getter and setter for the new attribute
+[step]
+--
+changeFile("cobigenexample/core/src/main/java/com/example/application/cobigenexample/customermanagement/dataaccess/api/CustomerEntity.java", { "file": "files/Placeholder.java", "placeholder": "private int age;" })
```

Comment:
I think you could simply regenerate with cobigen and also selekt entity infrastructure. This should also create getter and setters as far as I know.

Reply:
Please give it T least a try as this would even demonstrate the power of cobigen to adapt already existing files

Reply:
I just tried to do the cobigen cli tutorial without inserting the setters and getters manually. I added the 'devon cobigen adapt-templates' command to make 'cobigen generate' work. It seems that the getter and setter functions weren't generated automatically even thought entity infrastructure was selected[6].